src/sentry/options/defaults.py

```diff
@@ -128,6 +128,9 @@
 register("slack-v2.client-secret", flags=FLAG_PRIORITIZE_DISK)
 register("slack-v2.signing-secret", flags=FLAG_PRIORITIZE_DISK)
 
+# Slack Migration incremental rollout rate. Tied to feature handlers in getsentry
+register("slack.migration-rollout-rate", default=0, flags=FLAG_PRIORITIZE_DISK)
```

Comment:
@MeredithAnya How come we do this in sentry instead of getsentry since it's not actually used in Sentry?

Reply:
@scefali I was following how the `"discover2.rollout-rate"` was used, so mainly was just keeping it consistent. I don't know if there is more explanation as to 'why', @markstory would you happen to know?

Reply:
I added the option in sentry as that was the prior art I had seen before, and no one raised an issue when I opened the pull request. Looking again there are a few options registered in `getsentry/models/__init__.py`. If you wanted to keep everything together in getsentry that would work.